core/src/main/java/com/google/errorprone/bugpatterns/time/PreferJavaTimeOverload.java

```diff
@@ -112,6 +112,7 @@
 
   private static final Matcher<ExpressionTree> IGNORED_APIS =
       anyOf(
+          staticMethod().onClass("org.jooq.impl.DSL").named("inline"),
```

Comment:
Out of curiosity, any specific reason you didn't compare return types taking generics into account? (i.e. in this specific case, treating `Param<Long>` and `Param<Instant>` as different return types, such that `DSL.inline(long)` and `DSL.inline(Instant)` would no longer be considered overloads of each other)